apis/Google.Cloud.BigQuery.V2/Google.Cloud.BigQuery.V2/GetTableIamPolicyOptions.cs

```diff
@@ -0,0 +1,46 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License").
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at 
+//
+// https://www.apache.org/licenses/LICENSE-2.0 
+//
+// Unless required by applicable law or agreed to in writing, software 
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and 
+// limitations under the License.
+
+using Google.Apis.Bigquery.v2.Data;
+
+
+namespace Google.Cloud.BigQuery.V2;
+
+/// <summary>
+/// Options for <c>GetTableIamPolicy</c> operations.
+/// </summary>
+public sealed class GetTableIamPolicyOptions
+{
+    /// <summary>
+    /// Optional. The maximum policy version that will be used to format the policy.
+    /// </summary>
+    /// <remarks>
+    /// Valid values are 0, 1, and 3. Requests specifying an invalid value will be rejected.
```

Comment:
Potentially create our own enum here? This is really horrible...

Reply:
Yes, I thought about adding our enum, but I was worried that then we'd need to play catch up with new versions. Since we are exposing the Apiary types, we shouldn't need to make any changes in the future.

Unless we add the enum, but don't validate it's value, in which case we'd have the workaround of assigning the cast int to the enum property? What do you think?

Reply:
Mmm. Maybe it's best leaving it as it is.

Reply:
Agreed, at worst we can add an enum later on and sync the properties while we deprecate this one. Also maybe @shollyman has some context on how often this could change etc.

Reply:
This is the IAM meta API, so I have no confidence in anticipating changes.  The IAM folks may be better able to forecast here.

Reply:
OK, leaving as is for now, we can always reconsider an enum later on, but it seems like the raw int value will mean we'll be up to date easier.